README.md

```diff
@@ -1,24 +1,74 @@
-# README
+# Hello rails back end:
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+A Rails project as API.
 
-Things you may want to cover:
+[Link to the front end project PR](https://github.com/PolinaStamenova/hello-rails-front-end-project/pull/2)
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby
+- Rails
+- PostgreSQL
 
-* Configuration
+## Getting Started
 
-* Database creation
+To get a local copy up and running follow these simple example steps.
 
-* Database initialization
+## Instructions
 
-* How to run the test suite
+```
+$ cd <folder>
+```
 
-* Services (job queues, cache servers, search engines, etc.)
+```
+$ git clone git@github.com:PolinaStamenova/hello-rails-back-end.git
+```
 
-* Deployment instructions
+```
+$ cd hello-rails-back-end
+```
 
-* ...
+## Install
+
+```
+$ bundle install
+```
+
+## Run
+
+```
+$ rails db:create
+```
+
+```
+$ rails db:migrate
+```
+
+```
+$ rails db:seed
+```
+
+```
+rails s
+```
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.

Reply:
Thank you :)